rekise/vrx
# SPII-74: Integrate robot_localization package

### Changes: 

- Change gps frequency to `10 HZ`
- Change IMU configuration to `ENU`
- Integrate `robot_localization` package
- Update rviz config file to use `odom` as the fixed frame
- Remove existing tf broadcaster from vrx
- Create package `rekise_pkgs/vrx_launch` to contain consolidated launch file
- SPII-76: Publish robot's path and make it visible in rviz

### Extra Install : 
`sudo apt install ros-humble-robot-localization -y`

### To Test:

1. Launch simulator (gazebo)
2. Launch rviz and localization package using consolidated launch file `ros2 launch vrx_launch vrx.launch.py`
3. **Verify:** The robot should be visible with floating motion.  We can give thrust to the left and right thrusters using `rqt` and visualize the motion of the robot in correct direction.
